docusaurus-docs/docusaurus.config.js

```diff
@@ -183,6 +183,11 @@ const config = {
           '⚠️ We are in the process of revamping our docs, some pages may be incomplete or inaccurate. ⚠️',
         isCloseable: false,
       },
+      docs: {
+        sidebar: {
+          hideable: true,
```

Comment:
Did you intend to hide the sidebar for the page you added? Why is this change in this PR?

Reply:
Yes, this is intentional. We don't have any more pages for the JLab extension (all usage actions are similar to conda-store-ui), so the sidebar will be empty - hence hiding it.

Reply:
xref: https://github.com/conda-incubator/conda-store/pull/752/files#r1504642431

There is actually no way to completely hide it, hence this option to make the experience slightly nicer.

Reply:
Ah, I think I see now. 

As far as I understand, technically this change isn't related to the other changes in this PR. My preference would be to put this change in a separate PR, just to keep things neat and tidy, but not going to block on it

Reply:
I'd actually prefer to include it here because this docs page is what prompted the update.